%YAML:1.0
---
image_width: 1920
image_height: 1080
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 1.0140282933445519e+03, 0., 9.5079498948372941e+02, 0.,
       1.0112035618869359e+03, 5.0158220026880565e+02, 0., 0., 1. ]
distortion_coefficients: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -7.2598328944199111e-02, 1.7120699553544164e-01,
       -1.9861306279997147e-03, -5.8318690627218754e-03,
       -1.4833834191952294e-01 ]
